src/components/UserSelector/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { DropdownItem } from './DropdownItem';
+import { User } from '../../../types/User';
+
+interface DropdownMenuProps {
+  users: User[];
+  setIsActive: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+export const DropdownMenu: React.FC<DropdownMenuProps> = ({
+  users,
+  setIsActive,
+}) => {
+  return (
+    <div tabIndex={1} className="dropdown-menu" id="dropdown-menu" role="menu">
```

Comment:
Avoid using the same value for both `tabIndex` and `id` attributes. Consider using a more descriptive id value.